src/storage/wal_replayer.cpp

```diff
@@ -297,15 +294,11 @@ void WALReplayer::replayCopyNodeRecord(const kuzu::storage::WALRecord& walRecord
             if (wal->isLastLoggedRecordCommit()) {
                 return;
             }
-            auto catalogForRecovery = getCatalogForRecovery(DBFileType::ORIGINAL);
-            WALReplayerUtils::createEmptyDBFilesForNewNodeTable(
-                catalogForRecovery->getReadOnlyVersion()->getNodeTableSchema(tableID),
-                wal->getDirectory());
+            // TODO(Guodong): Do nothing for now. Should remove meta disk array and node groups.
```

Comment:
I will address these two TODOs for CopyNode, as we can apply special logic to do truncation of data and metadata files. Though I think the idea solution is to go through free pages instead of truncation (the todo for free pages is in issue https://github.com/kuzudb/kuzu/issues/1474). Free pages will also be the solution for DropTable and DropProperty, as we don't have existing mechanism to reclaim pages from a file.